setup.py

```diff
@@ -49,6 +49,7 @@
         'h5py>=2.10.0,<4',
         'pygam>=0.8.0,<1',
         'scikit-learn>=0.24.0,<2',
+        'gooey>=1.0.0,<2'
```

Comment:
This was intentionally not added, to allow the deeplc CLI to be easily installed without wxwidgets dependency, as this complicates the installation on Linux machines. To install deeplc with GUI, use `pip install deeplc[gui]]`.